Gemfile.lock

```diff
@@ -1,12 +1,12 @@
 GIT
-  remote: git://github.com/everypolitician/everypolitician-ruby.git
+  remote: https://github.com/everypolitician/everypolitician-ruby.git
```

Comment:
I've pushed some new commits, one to remove the unused `everypoliticianbot` gem (d60424d5d375ae8e993ec9bf7e186f07d5e996b0) and another to pin the version of `everypolitician-ruby` to the version we are using here (cb62ad6e6601a166ad6faa28285ab2d3621da8cf).